test/index.ts

```diff
@@ -30,7 +30,7 @@ async function runTests() {
     channels.push(makeAtomicsChannel());
   }
 
-  const client = new PyodideClient(() => new Worker());
+  const client = new PyodideClient<any>(() => new Worker());
```

Comment:
You set `T=any` in comsync, does that not naturally extend here? Maybe set it again for PyodideClient?